### PR TITLE
Signer

### LINKED ISSUE #75: Signer: Token field is cleared
On Unauthorized page you only have few seconds between pasting token and clicking authorize - otherwise the field is cleared.

### PR DESCRIPTION
**Pull request by goldylucks**
resolves #75 #76 
- [x] commit compiled files
- [x] build passing

### CODE REVIEW

File: signer/web/src/providers/appProvider.js [hunk 1]
```diff
@@ -14,14 +14,19 @@ export default class appProvider {
...
 
   onWsError () {
-    isParityRunning(this.wsPath)
-      .then(isRunning => {
-        this.store.dispatch(updateAppState({ isParityRunning: isRunning, isLoading: false, isWsConnected: false }));
-      });
+    this.store.dispatch(
+      updateAppState({ isLoading: false, isWsConnected: false })
+    );
+    // isParityRunning(this.wsPath)
```
Comment (tomusdrw) [inline signer/web/src/providers/appProvider.js:26]: Remove?

Comment (tomusdrw) [inline signer/web/src/providers/appProvider.js:26]: Or I'm pretty sure we already have a `ping` endpoint - it should be possible to check if parity is running or not.

Comment (goldylucks) [inline signer/web/src/providers/appProvider.js:26]: yup, would b done in a separate PR

Comment (goldylucks) [inline signer/web/src/providers/appProvider.js:26]: other than that, merge?

Comment (tomusdrw) [inline signer/web/src/providers/appProvider.js:26]: I don't see issue for that. Would prefer to have commented-out code removed.
